examples/flows/evaluation/eval-summarization/README.md

```diff
@@ -0,0 +1,106 @@
+# Summarization Evaluation
+
+This flow implements a reference-free automatic abstractive summarization evaluation across four dimensions: fluency, coherence, consistency, relevance. Each dimension uses a prompt to score a generated summary against the source document from which it was generated. Each dimension's prompt has been meta-evaluated against the [SummEval benchmark](https://arxiv.org/abs/2007.12626) and produces state-of-the-art scores in terms of correlation to expert human judgements.
+
+## Introduction
+
+Provide a detailed description of the flow, including its components, inputs, outputs, and any dependencies. Explain how the flow works and what problem it solves. This section should give users a clear understanding of the flow's functionality and how it can be used. -- TODO remove this when done
```

Comment:
Mark on this TODO

Reply:
thanks for your review! yeah the purpose of this PR was to test CI pipeline early actually. That's why we put [DO NOT MERGE, DRAFT] in the titile and kept multiple TODOs in readme we are still working on. It turns out forked repo's CI pipeline didn't have enough permission to run CI successfully so based on PG's recommendation, we moved to promptflow repo's branch and new PR is here. I'm closing this PR now as all the updates will come to that branch instead

https://github.com/microsoft/promptflow/pull/2037